src/components/notifications/Notifery.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div class="mb-3">
+        <label for="notifery-api-key" class="form-label">{{
+            $t("API Key")
+        }}</label>
+        <HiddenInput
+            id="notifery-api-key"
+            v-model="$parent.notification.notiferyApiKey"
+            :required="true"
+            autocomplete="new-password"
+        ></HiddenInput>
+    </div>
+
+    <div class="mb-3">
+        <label for="notifery-title" class="form-label">{{ $t("Title") }}</label>
+        <input
+            id="notifery-title"
+            v-model="$parent.notification.notiferyTitle"
+            type="text"
+            class="form-control"
+            :placeholder="$t('Uptime Kuma Alert')"
```

Comment:
Not actually translated in the backend and not in `en.json` => cannot be translated

```suggestion
            placeholder="Uptime Kuma Alert"
```